src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -99,6 +120,10 @@ public function registerContainerConfiguration(LoaderInterface $loader)
                 $kernelDefinition->addTag('kernel.event_subscriber');
             }
 
+            $container->fileExists($this->getProjectDir().'/config/bundles.php');
+            $container->setParameter('container.dumper.inline_class_loader', \PHP_VERSION_ID < 70400 || !ini_get('opcache.preload'));
```

Comment:
isn't it an issue to change the cache-warmup behavior based on the `preload` config, while you need to warmup your cache before being able to confire the preloading ?

Reply:
This will remove inline "require" statements that are generated currently.
They are not needed anymore when preloading is used.

Reply:
my issue is that running `cache:warmup` then adding the `preload` file in the php.ini and then running `cache:warmup` again will produce a different cache.

Reply:
yes ok, I wouldn't care much :)

Reply:
OK, removed :)

Reply:
Ppl that care about the extra perf this would provide would better enable preloading instead on PHP 7.4.
This will prevent issues with CLI vs FPM with different config for opcache.preload.